modules/acm/variables.tf

```diff
@@ -62,6 +62,11 @@ variable "create_ssh_key" {
   default     = true
 }
 
+variable "secret_type" {
```

Comment:
This variable should have a default.

Reply:
ok, provide the value to match before. more generally, we should generate this from the CRD